server/historian/packages/historian-base/src/routes/summaries.ts

```diff
@@ -68,5 +76,20 @@ export function create(
                 201);
     });
 
+    router.delete("/repos/:ignored?/:tenantId/git/summaries",
+    throttle(throttler, winston, commonThrottleOptions),
+    (request, response, next) => {
+        const softDelete = request.get("Soft-Delete")?.toLowerCase() === "true";
+        const summaryP = deleteSummary(
```

Comment:
Does the DELETE Summary API happen per document or for the whole tenant? If it's per document, does it learn the documentID through the Storage-Routing-ID header?

Reply:
Yes, the deletion is per document. Calls to Historian obtains the documentId from the auth token [here](https://github.com/microsoft/FluidFramework/blob/15d5b248622fc47ab4e8e919df86bb4349826338/server/historian/packages/historian-base/src/routes/utils.ts#L66-L71), and sets it in the Storage-Routing-Id header for calls to the underlying storage.

Reply:
Cool, I just wanted to confirm if you were indeed planning to use Storage-Routing-Id (which as you said, is created by also extracting Doc ID from the auth token) or if instead this route would be used to remove all summaries per tenant.